.github/workflows/auto-label-prs.yml

```diff
@@ -0,0 +1,19 @@
+name: Label new PRs
+on:
+  pull_request:
+    types: [opened,ready_for_review]
+
+jobs:
+  label-new-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Label new drafts
+        uses: andymckay/labeler@master
+        if: github.event.pull_request.draft == true
+        with:
+          add-labels: 'A3-inprogress'
+      - name: Label new PRs
+        uses: andymckay/labeler@master
+        if: github.event.pull_request.draft == false
+        with:
+          add-labels: 'A0-pleasereview'
```

Comment:
`A3-inprogress` isn't removed here, shouldn't it?

Reply:
Good spot! Went missing somehow